vignettes/tutorial-create-your-first-rhino-app.Rmd

```diff
@@ -111,15 +126,17 @@ If you want to know more about it, check [this](https://appsilon.github.io/rhino
 Now, once you are all set up, let's run it:
   
 ```r
-# In R console
-
 shiny::shinyAppDir(".")
 ```
 
 And here is what you should be seeing right now:
   
+<div class='app-preview'>
+  
 ![](images/basic_tutorial/init_application.png)
 
+</div>
+
```

Comment:
```suggestion
<div class="app-preview">
  ![](images/basic_tutorial/init_application.png)
</div>
```
1. Here and in all subsequent lines I'd recommend forgo blank lines and indent the contents of the `div`.
2. I'd go with double-quotes (`"`) for HTML attributes. We don't have a style guide for HTML really, but it seems to be a widely-adopted standard.